do-not-track.rst

```diff
@@ -0,0 +1,63 @@
+.. post:: June 8, 2018
+   :tags: privacy, advertising
+   :author: David
+   :location: SAN
+
+Do Not Track at Read the Docs
+=============================
+
+Today, we are pleased to announce that Read the Docs honors `Do Not Track`_ (DNT).
+DNT is a browser preference that requests that a user not be tracked
+across the internet while browsing the web.
+
+While there isn't a consensus on precisely what DNT should mean,
+we are following the Electronic Frontier Foundation's (EFF) `guidelines`_
+for Do Not Track as we believe that gives a good balance
+between the privacy expectations of users and the reality of running a business
+and keeping Read the Docs sustainable.
+
+.. _Do Not Track: https://allaboutdnt.com/
+.. _guidelines: https://www.eff.org/issues/do-not-track
+
+
+What DNT means for our advertising
+----------------------------------
+
+Advertising at Read the Docs was already built with privacy in mind.
+With our `Ethical Ads`_, we previously committed to not tracking users,
+not selling user data, and hosting ads ourselves.
+
+Our support for Do Not Track does not change that.
+Instead, it formalizes our commitment
+to be the same as guidelines produced by the EFF and elsewhere.
+
+At Read the Docs this means:
+
+* We **do not** do behavioral ad targeting regardless of your DNT preference.
+* Our logs that store any personal data are deleted after no more than ten days.
+  We do this for all users even those who don't have DNT enabled.
+* When a user has DNT enabled, we do not send data to our analytics.
```

Comment:
Feels like this should be above the "ads" section -- as it applies outside the ad code. Perhaps a "What we implemented" section before the "what it means for ads"?